package.json

```diff
@@ -16,10 +16,10 @@
     "addon_runners": "node ./test/run.js addon_runners",
     "unit": "node ./test/run.js unit",
     "documentation": "mocha -t 5000 test/documentation",
-    "jshint": "jshint bin/jpm lib test --exclude test/addons,test/addon_runners,test/fixtures",
-    "jscs": "jscs bin/jpm lib",
+    "eslint": "eslint bin/jpm lib test",
```

Comment:
I think the `lint` command is enough, right? I'd like to avoid aliases if we don't need them.